shared/teams/edit-team-description/index.tsx

```diff
@@ -1,57 +1,100 @@
 import React from 'react'
 import * as Kb from '../../common-adapters'
 import * as Styles from '../../styles'
+import * as TeamsGen from '../../actions/teams-gen'
+import * as Container from '../../util/container'
+import * as Constants from '../../constants/teams'
+import * as Types from '../../constants/types/teams'
 
-export type Props = {
-  onSubmit: (description: string) => void
-  onClose: () => void
-  origDescription: string
-  teamname: string
-  waitingKey: string
-}
+type Props = Container.RouteProps<{teamID: Types.TeamID}>
 
 const EditTeamDescription = (props: Props) => {
-  const {origDescription, teamname, onClose, onSubmit, waitingKey} = props
+  const teamID = Container.getRouteProps(props, 'teamID', Types.noTeamID)
+
+  const teamname = Container.useSelector(state => Constants.getTeamNameFromID(state, teamID))
```

Comment:
likely easier to just pull state out vs calling useselector multiple times

Reply:
As discussed in person, going to leave this as is.